### ISSUE TITLE
Add a new word

### ISSUE THREAD

**Issue by Showndarya** [Showndarya/Hacktoberfest#25]
Add a new word to the repository.

Create a JSON file using the following example as format:

```json
{
    "word": "Beautiful",
    "definitions": [
        "pleasing the senses or mind aesthetically",
        "of a very high standard; excellent"
    ],
    "parts-of-speech": "Adjective"
}
```

Name the file as <word-name>.json and save it in the respective repository corresponding to the starting letter of the word. Create a folder if one doesn't exits.

Wait for the pull request to merge and get featured in the [Hall of Fame](https://showndarya.github.io/Hacktoberfest/Contributors_Page_Source/)  :sunglasses:

**Comment by SrVladyslav** [comment]
Can I do it?

**Comment by Showndarya** [comment]
Sure. Create a pull request and add a reference to this issue.

Start the pull request description with,

Issue #25

**Comment by cynthiatayloronline** [comment]
Hi, Can more than one person contribute to this issue?

**Comment by Showndarya** [comment]
@cynthiatayloronline Yes, surely. You can contribute to this repository with more words too.

**Comment by PritishSehzpaul** [comment]
Hi I would be adding some words to the repo. Please review and merge the PR.

**Comment by kalpanaT-05** [comment]
hi @Showndarya can i do this?

**Comment by Tahreem-Khalil** [comment]
**hey kindly review my pull request #2876 thank you.**
